ui/components/app/desktop-enable-button/desktop-enable-button.component.js

```diff
@@ -40,6 +43,10 @@ export default function DesktopEnableButton() {
     if (desktopEnabled) {
       await disableDesktop();
       setDesktopEnabled(false);
+      trackEvent({
+        category: EVENT.CATEGORIES.DESKTOP,
+        event: `Disable Desktop Button Clicked`,
```

Comment:
instead of separate events for enable/disabled, these should be properties of the same event.  another option is that this is actually a setting change.

either of these works:
```
event name: Desktop Button Clicked
properties: {enabled: true|false}
```
or
```
event name: Settings Updated
properties: {desktop_enabled: true|false}
```

Reply:
I've opted for the settings change as you suggested:
```
event name: Settings Updated
properties: {desktop_enabled: true|false}
```